sycl/test-e2e/format.py

```diff
@@ -154,10 +154,14 @@ def execute(self, test, litConfig):
         if isinstance(script, lit.Test.Result):
             return script
 
+        unsplit_test = False
```

Comment:
I don't know why, but something in me has a really huge dislike towards "unsplit" anywhere in regard to this feature...

Reply:
Changed it to `ignore_line_filtering`, is that better?